packages/compass-components/src/components/error-warning-summary.tsx

```diff
@@ -18,6 +18,11 @@ const listStyle = css({
   listStylePosition: 'inside',
 });
 
+const summaryStyles = css({
+  display: 'flex',
+  justifyContent: 'space-between',
```

Comment:
You can keep it as-is, but just worth keeping in mind that space-between is a brittle way to push something to the end of the container as having more flex children inside the component will result in possibly unexpected spacing applied to them. If the intention is not to evenly space multiple flex children but to push last one to the end, it's always a bit more robust to do it with `margin-left: auto`